docs/data/base/components/badge/badge.md

```diff
@@ -94,7 +94,7 @@ The following code snippet applies a CSS class called `my-badge` to the badge sl
 import { useBadge } from '@mui/base/BadgeUnstyled';
 ```
 
-The `useBadge` hook lets you apply the functionality of `BadgeUnstyled` to a fully custom component.
+The `useBadge` hook lets you apply the functionality of the Unstyled Badge to a fully custom component.
```

Comment:
Yeah, I was a little concerned about overusing "Unstyled [Component].". I could be a little more liberal about cutting "Unstyled" when it's redundant or obvious that we're referring to the Base component. I'll remove it from this section and do another pass through the docs to see if there are other places where it could be removed.